### PR TITLE
Issue 1467 - Audit web page and recipe page outlines that include the DOM

### PR DESCRIPTION
**Pull request by ChasVanDav**
# Background
Issue# 1467 tasked contributors to separate the DOM lesson plan into separate lessons. Learning about the DOM and how it works can require more processing time and practice for new engineers, so breaks were deliberately built in to allow for time and space to process the information.
# Purpose
to audit two outlines that include learning and practicing the DOM manipulation
# Source
The audited files can be found below:
[Web Outline](https://github.com/Techtonica/curriculum/blob/main/web/js-dom-methods.md)
[Recipe Page Outline](https://github.com/Techtonica/curriculum/blob/main/projects/recipe-page/phase-4-DOM-Manipulation.md)
# Checks & Key Changes
Web Outline
-  checked all links, confirmed to all be up-to-date
-  modified some grammar and awkward phrasing

Recipe Page Outline
- updated to update 'var' to 'let'
- added note to screenshot containing 'var' explaining its deprecation and alternative options
- fixed typo

n.b. There was an errant file that made its way into this pr and was subsequently removed. Kindly ignore that!

### CODE REVIEW

File: practice/practice-the-pr-review-process.md
Comment (ChasVanDav) [inline practice/practice-the-pr-review-process.md:1]: please ignore this file from a separate pr

File: projects/recipe-page/phase-4-DOM-Manipulation.md
Comment (ChasVanDav) [inline projects/recipe-page/phase-4-DOM-Manipulation.md:1]: added note on var deprecation to supplement screenshot of code that uses var for one of the variables.

**Comment by ChasVanDav** [comment]
@daaimah123 @thisIsTheWei @mai-repo Please take a look at the changes I made for the audit of two outlines related to the DOM. TIA!

**Review by daaimah123 — APPROVED** [review]

**Comment by ChasVanDav** [comment]
@daaimah123 To echo our conversation over Slack, this pr will be reviewed by @thisIsTheWei on Monday (12/30) and will be ready for your final review! @mai-repo feel free to jump in anytime between now and Monday/Tuesday. Thank you!

File: projects/recipe-page/phase-4-DOM-Manipulation.md [hunk 1]
```diff
@@ -44,7 +44,9 @@ document.querySelector('ul').appendChild(newListItem);
 
 #### 2. Adding a check mark as an element to a list
 
-To add checkbox to the page, we’ll need to use a this process
+To add checkbox to the page, we’ll need to use this process:
+
+(n.b. 'var' is now deprecated, so use either 'let' or 'const' to create a variable)
```
Comment (thisIsTheWei) [inline projects/recipe-page/phase-4-DOM-Manipulation.md:49]: I believe `var` is not technically deprecated in JavaScript, but it's strongly recommended to use let and const instead in modern JavaScript development. Should we describe it in this way?

Comment (ChasVanDav) [reply projects/recipe-page/phase-4-DOM-Manipulation.md:49]: I will adjust the note to explain that it is not deprecated for backward compatibility reasoning but it is now advised to use let or consst. Thank you @thisIsTheWei !!

Comment (ChasVanDav) [reply projects/recipe-page/phase-4-DOM-Manipulation.md]: https://github.com/Techtonica/curriculum/pull/2193/commits/21662f9c5ebcb94c7ce551a068c5ef50fa869779 @thisIsTheWei

**Review by thisIsTheWei — APPROVED** [review]

File: projects/recipe-page/phase-4-DOM-Manipulation.md (hunk 1 above)
Comment (thisIsTheWei) [reply projects/recipe-page/phase-4-DOM-Manipulation.md]: Great work!

**Review by mai-repo — APPROVED** [review]
@ChasVanDav The changes looks good to me!

**Comment by ChasVanDav** [comment]
@daaimah123 All reviewers have approved this pr. I will now merge this pr and close the issue!

**Pull request merged**